bindings/java/Cargo.toml

```diff
@@ -32,8 +32,8 @@ crate-type = ["cdylib"]
 doc = false
 
 [features]
-# Enable all opendal default feature by default.
 default = [
+  # Enable all opendal default feature by default.
```

Comment:
Maybe redundant?

Reply:
It existed previously. Let's review this in another thread.

Reply:
But I touch it once more so change together :D